Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -69,15 +73,26 @@ def audit
       files = ARGV.resolved_formulae.map(&:path)
     end
 
+    only_cops = ARGV.value("only-cops").to_s.split(",")
```

Comment:
Initially I was trying to do that. The code would remain almost the same, but for starters I pushed this code for review. That would also conflict with the current convention that values of `--only` would be translated to `audit_<method_name>` . I mean, it may or may not look **clean**  :-|